fetch.bs

```diff
@@ -2227,11 +2224,8 @@ run these steps:
  <li><p>Let <var>port</var> be <var>url</var>'s
  <a for=url>port</a>.
 
- <li><p>If <var>scheme</var> is "<code>ftp</code>" and <var>port</var> is 20 or 21, then
- return <b>allowed</b>.
-
- <li><p>Otherwise, if <var>scheme</var> is a <a>network scheme</a> and
- <var>port</var> is a <a>bad port</a>, then return <b>blocked</b>.
+ <li><p>If <var>scheme</var> is an <a>HTTP(S) scheme</a> and <var>port</var> is a <a>bad port</a>,
```

Comment:
Optional: now that scheme and port are only referenced once, they don't benefit that much from being factored out into variables (i.e. you could say "if url's scheme is... and url's port is ...").